Cargo.lock

```diff
@@ -619,7 +619,7 @@ dependencies = [
  "clippy_utils",
  "if_chain",
  "itertools 0.10.1",
- "pulldown-cmark",
+ "pulldown-cmark 0.8.0",
```

Comment:
Hmm seems like maybe we should update mdbook and clippy's dependencies first to avoid duplicates.

Reply:
Good idea!

Reply:
Do you want to open PRs for them?

Reply:
I did and I learned that for clippy it needs to be done in here first.